API.md

```diff
@@ -138,6 +187,38 @@ curl https://host/projects/00000000-0000-0000-0000-000000000000
 }
 ```
 
+### get-item-comment
```

Comment:
Is this endpoint necessary? I don't see any cases where we need to request individual comments. It will be simpler on the frontend to always request the list of comments.